### DISCUSSION TITLE
How to disable "Error - continue" pop-up?

### DISCUSSION THREAD

**Discussion by HubKing** [microsoft/vscode-java-debug#1304] [Q&A]
I do not want to see this pop-up at all. The behaviour I want is "always NOT continue" when there is a build error. This pop-up does not automatically disappear even after I fixed the code and build has succeeded; I must manually close it. Very annoying. How to make this pop-up not appear?

![image](https://user-images.githubusercontent.com/10900489/222882406-023dffe3-3de2-41f1-99a0-30d26f6e15ea.png)

**Comment by testforstephen** [comment]
I don't think it's a good behavior to silently fail the debugging on a build error. If you don't want a pop-up during build error, then it must provide some alternative ways to hint user about the situation.

**Reply by HubKing** [reply]
It doesn't have to "silently fail". I can see the errors in the error panel, and this extension could just automatically switch to the error panel instead of showing the pop-up that needs to be closed.

That behaviour is what I can already achieve in Visual Studio. See, VS already has the "never show again" option.

![image](https://user-images.githubusercontent.com/10900489/223078900-f3ea76d6-fb39-4d6f-a71f-67a9cf46ebcb.png)
